rootfs/scheduler/resources/deployment.py

```diff
@@ -237,6 +237,12 @@ def in_progress(self, namespace, name, timeout, batches, replicas, tags):
             self.log(namespace, 'Deploy operation for Deployment {} in has expired. Rolling back to last good known release'.format(name), level='DEBUG')  # noqa
             return False, True
 
+        try:
+            self._check_for_failed_events(namespace, labels=labels)
```

Comment:
If we not checked failures in RS here, use will be get in progress status of build, during default of custom deploy timeout, fo example:
```
deis scale cmd=4
error: Quota exceeded
deis scale cmd=3
Another build in progress
deis scale cmd=3
Another build in progress